lib/services/large_result_set.js

```diff
@@ -30,8 +30,8 @@ function LargeResultSetService(connectionConfig, httpClient)
   {
     return err && (
       err.code === 'ECONNRESET' || err.code === 'ETIMEDOUT' ||
```

Comment:
Can we split it into two lines? Each alternative branch in a new line.

Reply:
yes, we can